networking/ingress-operator.adoc

```diff
@@ -56,6 +56,8 @@ include::modules/nw-using-ingress-forwarded.adoc[leveloffset=+2]
 
 include::modules/nw-http2-haproxy.adoc[leveloffset=+2]
 
+include::modules/nw-ingress-controller-proxy-protocol.adoc[leveloffset=+2]
```

Comment:
The include isn't building, just a lil typo: 
include::modules/nw-ingress-controller-configuration-proxy-protocol.adoc[leveloffset=+2]